### PR TITLE
[12.0] foreign customers

### PR DESCRIPTION
**Pull request by eLBati**
TODO
Rifarla con i commit di https://github.com/OCA/l10n-italy/pull/888

### CODE REVIEW

**Comment by sergiocorato** [comment]
@eLBati ho aggiunto il tuo commit con alcuni miglioramenti [vedi](https://github.com/OCA/l10n-italy/pull/861/commits/b1fa561ddf9ce266865c3588c17dc8d839d22647)

**Comment by eLBati** [comment]
@sergiocorato grazie, inclusi

File: l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py [hunk 1]
```diff
@@ -345,7 +345,21 @@ def setCedentePrestatore(self, company, fatturapa):
     def _setDatiAnagraficiCessionario(self, partner, fatturapa):
         fatturapa.FatturaElettronicaHeader.CessionarioCommittente.\
             DatiAnagrafici = DatiAnagraficiCessionarioType()
-        if not partner.vat and not partner.fiscalcode:
+        if (
+            not partner.vat and not partner.fiscalcode and
+            partner.codice_destinatario == 'XXXXXXX' and
+            partner.country_id.code
+        ):
+            # SDI accpets missing VAT# for foreign customers by setting a fake
```
Comment (tafaRU) [inline l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py]: s/accpets/accepts

**Comment by eLBati** [comment]
moving to https://github.com/OCA/l10n-italy/pull/901

**Pull request closed without merging**